src/Herbs.js

```diff
@@ -0,0 +1,28 @@
+import React, { Component } from 'react';
+import request from 'superagent';
+import List from './List.js';
+
+export default class Herbs extends Component {
+    state = {
+        loading: false,
+        herbs: []
+    }
+
+    async componentDidMount() {
+        // const data = await request.get('https://still-plateau-72252.herokuapp.com/herbs');
+        const data = await request.get('http://localhost:5555/herbs');
```

Comment:
this needs to be pointing to your heroku app for submission.  I won't be running your backend for grading